usagov/usagov-2021
# Usagov 1822 callout weight

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1822

## Description

Removes the font-weight overriding the default for call-out description.

## Type of Changes

![image](https://github.com/user-attachments/assets/9e0b5c33-c385-44fd-9e0e-c1003ea7f451)

![image](https://github.com/user-attachments/assets/adbc30f5-1a71-480e-a339-cc8bfc1ba0f4)


## Testing Instructions


Need benefits search blocks enabled on homepage

### Change Requirements



### Validation Steps

- Confirm font-weight is 400 for the text below the title in the call-out (I had to tick the "browser styles" box in dev tools inspector)

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
